### PR TITLE
Adding alert to pages that reference grouparoo generate

### PR DESCRIPTION
**Pull request by krishnaglick**
## Change description

Adding alert to pages that reference grouparoo generate

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Comment by krishnaglick** [comment]
@bleonard I'm using the `warning` alert style, unsure if that's what we want to go with.

**Review by teallarson — CHANGES_REQUESTED** [review]
Mentioning that we've improved the experience is awesome.

I think noting that the steps in the post will not be reproducible is important.  The `grouparoo generate` command used in these posts will be fully deprecated in 0.8.1 (Presumably?  It's coming up soon here in tracker).

File: pages/blog/a-single-source-of-truth.mdx [hunk 1]
```diff
@@ -7,6 +7,16 @@ image: "google-sheets-source/source-data-preview.png"
 tags: [connections, engineering]
 ---
 
+<Alert variant="warning">
+  We've improved the{" "}
+  <a href="https://www.grouparoo.com/docs/getting-started">Getting Started</a>{" "}
+  Experience! Check out our{" "}
+  <a href="https://www.grouparoo.com/docs/config">UI Configuration</a> method.
+  <br />
+  The steps utilizing <code>grouparoo generate</code> will not be replicatible
```
Comment (teallarson) [inline pages/blog/a-single-source-of-truth.mdx]:
```suggestion
  The steps utilizing <code>grouparoo generate</code> will not be replicable
```

File: pages/blog/using-sync-modes.mdx [hunk 2]
```diff
@@ -7,6 +7,16 @@ image: "using-sync-modes/210505-sync-modes.png"
 tags: [engineering, product]
 ---
 
+<Alert variant="warning">
+  We've improved the{" "}
+  <a href="https://www.grouparoo.com/docs/getting-started">Getting Started</a>{" "}
+  Experience! Check out our{" "}
+  <a href="https://www.grouparoo.com/docs/config">UI Configuration</a> method.
+  <br />
+  The steps utilizing <code>grouparoo generate</code> will not be replicatible
```
Comment (teallarson) [inline pages/blog/using-sync-modes.mdx]:
```suggestion
  The steps utilizing <code>grouparoo generate</code> will not be replicable
```

File: pages/blog/operational-analytics.mdx [hunk 3]
```diff
@@ -10,6 +10,16 @@ image: "operational-analytics/unsplash-pipes.jpg"
 tags: [engineering, data]
 ---
 
+<Alert variant="warning">
+  We've improved the{" "}
+  <a href="https://www.grouparoo.com/docs/getting-started">Getting Started</a>{" "}
+  Experience! Check out our{" "}
+  <a href="https://www.grouparoo.com/docs/config">UI Configuration</a> method.
+  <br />
+  The steps utilizing <code>grouparoo generate</code> will not be replicatible
```
Comment (teallarson) [inline pages/blog/operational-analytics.mdx]:
```suggestion
  The steps utilizing <code>grouparoo generate</code> will not be replicable
```

**Review by teallarson — APPROVED** [review]
Just one spelling tweak and we're good!

**Pull request merged**